Magento2/Tests/Less/ColonSpacingUnitTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Tests\Less;
+
+class ColonSpacingUnitTest extends AbstractLessSniffUnitTestCase
+{
+    /**
+     * @inheritdoc
+     */
+    public function getErrorList()
+    {
+        return [
+            8 => 1,
+            12 => 1,
+            16 => 2,
```

Comment:
I believe there should be an error on line 20 as well

Reply:
Absolutely, good catch!

Reply:
@sivaschenko the fix for the newline checking is harder than expected, as PHPCS does not respect newlines in-between instructions, so there is no way we can check this case with the current code. Should I put this as a new issue?

Reply:
I've created https://github.com/magento/magento-coding-standard/issues/242